shared/fs/common/path-item-action/menu.tsx

```diff
@@ -160,7 +160,8 @@ const makeMenuItems = (props: Props, hideMenu: () => void) => {
       ? [
           {
             danger: true,
-            onClick: hideMenuOnClick(props.ignoreTlf, hideMenu),
+            disabled: props.ignoreTlf === 'disabled',
+            onClick: props.ignoreTlf === 'disabled' ? undefined : hideMenuOnClick(props.ignoreTlf, hideMenu),
```

Comment:
I feel like disabled conveys a different meaning than a spinner + disabled, and users will be confused without the spinner "why can't I ignore this folder??". How much effort do you think it would be to finagle a spinner into this menu?

Reply:
I mean that when I open the menu, there should be a spinner on the disabled ignore button, to show that i can't ignore because something is in progress.

Reply:
Sounds good; will do!